doc/sailfish-webview-webviewexperimental.qdoc

```diff
@@ -0,0 +1,181 @@
+/****************************************************************************
+**
+** Copyright (c) 2021 Open Mobile Platform LLC
+**
+****************************************************************************/
+
+/*!
+\page webview-experimental.html
```

Comment:
I was thinking that rather saying this as "experimental" maybe this could called "advanced" and have there those rights to modify in place. How do you feel it @chriadam @llewelld ?

Reply:
Both "advanced" and "experimental" are clear, in my opinion.

Perhaps "Experimental" gives the impression that they may not work as expected, so I'd say it depends whether that's the case or not. If they're called "Advanced" then I think devs are more likely to use them (which may or may not be a good thing).

Reply:
We can change "experimental" to "advanced" in future without ruffling feathers, so I'd be inclined to leave it as "experimental" for now and potentially consider promoting them in future.